QUICKSTART.md

```diff
@@ -0,0 +1,279 @@
+.
+└── root
+    ├── compute
+    └── espw
+```
+
+## 2. Example deployment of Apache HTTP Server workload into two kind local clusters
+
+### a. Stand up a local florin and guilder kind clusters
+
+Create the first edge cluster:
+
+```shell
+kind create cluster --name florin --config examples/florin-config.yaml
+```  
+
+Create the second edge cluster:
+
+```shell
+kind create cluster --name guilder --config examples/guilder-config.yaml
+```  
+
+### b. Create a KubeStellar Inventory Management Workspace (IMW) and Workload Management Workspace (WMW)
+
+IMW are used by KubeStellar to store inventory objects (sync targets and placement). Create an IMW named `example-imw` with the following command:
+
+```shell
+kubectl config use-context root
+kubectl ws root
+kubectl ws create "example-imw"
+```
+
+WMW are used by KubeStellar to store workloads and edge placement objects. Create an WMW named `example-wmw` in a `my-org` workspace with the following command:
+
+```shell
+kubectl ws root
+kubectl ws create "my-org" --enter
+kubectl kubestellar ensure wmw example-wmw
+```
+
+### c. Onboarding the clusters
+
+Let's begin by onboarding the `florin` cluster:
+
+```shell
+kubectl kubestellar prep-for-cluster --imw root:example-imw florin  env=prod
+```
+
+which should yield something like:
+
+```console
+Current workspace is "root:example-imw".
+synctarget.workload.kcp.io/florin created
+location.scheduling.kcp.io/florin created
+synctarget.workload.kcp.io/florin labeled
+location.scheduling.kcp.io/florin labeled
+Current workspace is "root:example-imw".
+Current workspace is "root:espw".
+Current workspace is "root:espw:9nemli4rpx83ahnz-mb-c44d04db-ae85-422c-9e12-c5e7865bf37a" (type root:universal).
+Creating service account "kcp-edge-syncer-florin-1yi5q9c4"
+Creating cluster role "kcp-edge-syncer-florin-1yi5q9c4" to give service account "kcp-edge-syncer-florin-1yi5q9c4"
+
+ 1. write and sync access to the synctarget "kcp-edge-syncer-florin-1yi5q9c4"
+ 2. write access to apiresourceimports.
+
+Creating or updating cluster role binding "kcp-edge-syncer-florin-1yi5q9c4" to bind service account "kcp-edge-syncer-florin-1yi5q9c4" to cluster role "kcp-edge-syncer-florin-1yi5q9c4".
+
+Wrote physical cluster manifest to florin-syncer.yaml for namespace "kcp-edge-syncer-florin-1yi5q9c4". Use
+
+  KUBECONFIG=<pcluster-config> kubectl apply -f "florin-syncer.yaml"
+
+to apply it. Use
+
+  KUBECONFIG=<pcluster-config> kubectl get deployment -n "kcp-edge-syncer-florin-1yi5q9c4" kcp-edge-syncer-florin-1yi5q9c4
+
+to verify the syncer pod is running.
+Current workspace is "root:example-imw".
+Current workspace is "root".
+```
+
+An edge syncer manifest yaml file is created in your current director: `florin-syncer.yaml`. The default for the output file is the name of the SyncTarget object with “-syncer.yaml” appended.
+
+Now le's deploy the edge syncer to the `florin` edge cluster:
+
+  
+```shell
+KUBECONFIG=$florin_kubeconfig kubectl apply -f florin-syncer.yaml
```

Comment:
A user who has done what she was told does NOT _have_ a "florin_kubeconfig" at this point.  Instead, the `kind create cluster` commands added _contexts_ to whatever kubeconfig was being used at that time.  So a user following these instructions needs to use the following command instead:

```shell
kubectl --context kind-florin apply -f florin-syncer.yaml
```

Reply:
Fixed